draft-ietf-tls-tls13.md

```diff
@@ -1395,7 +1395,7 @@ Structure of this message:
        uint8 CipherSuite[2];    /* Cryptographic suite selector */
 
        struct {
-           ProtocolVersion max_supported_version = { 3, 4 };    /* TLS v1.3 */
+           ProtocolVersion legacy_version = { 3, 3 };    /* TLS v1.2 */
```

Comment:
Registry and uint16 SGTM. A registry will be useful anyway to burn things like private use values for drafts and GREASE values if the WG is so inclined.